lib/engine/game/g_1860.rb

```diff
@@ -388,7 +388,7 @@ def check_nationalize?
       end
 
       def get_or_revenue(info)
-        info.dividend.kind == 'withhold' ? 0 : info.revenue
+        !info.dividend.is_a?(Action::Dividend) || info.dividend.kind == 'withhold' ? 0 : info.revenue
```

Comment:
What problem is this solving? Why wouldn't a dividend not have a dividend action?

Reply:
When the Dividend step is skipped (ie, no train) the OperatingInfo gets the previous action so that the links on the spreadsheet work (see the mentioned issue)

Reply:
But the amount isn't 0 in this case. It's the revenue the leased train runs for.

Reply:
you can jump to the action in the spreadsheet

Reply:
How? Where is that action stored? It just looks like a dangling object

Reply:
@roseundy @daniel-sousa-me can you guys clarifiy this particular line? why is the dividend not an action::dividend, what type is it?

Reply:
It's whatever @game.actions.last is on line 30 of step/1860/dividend.rb

Reply:
Yeah, it's the action that triggered a skip on the Dividends

Reply:
This is not specific to 1860 at all. It is also changed in `step/dividend.rb`. 1860 just was problematic because I didn't notice it was completely bypassing the parent code

Reply:
Hmm, pretty sure I borrowed that code from a different game.